lib/i18n.rb

```diff
@@ -231,6 +231,31 @@ def translate!(key, **options)
     end
     alias :t! :translate!
 
+    # Returns an array of interpolation keys for the given translation key
+    #
+    # Suppose we have the following:
+    #   I18n.t 'example.zero' == 'Zero interpolations'
+    #   I18n.t 'example.one' == 'One interpolation %{foo}'
+    #   I18n.t 'example.two' == 'Two interpolations %{foo} %{bar}'
+    #   I18n.t 'example.one', locale: :other == 'One interpolation %{baz} %{bar}'
+    #
+    # Then we can expect the following results:
+    #   I18n.interpolation_keys('example.zero') #=> []
+    #   I18n.interpolation_keys('example.one') #=> ['foo']
+    #   I18n.interpolation_keys('example.two') #=> ['foo', 'bar']
+    #   I18n.interpolation_keys('one', scope: 'example', locale: :other) #=> ['baz']
+    #   I18n.interpolation_keys('does-not-exist') #=> []
```

Comment:
...Or should it be `nil` 🤷🏼‍♂️

Reply:
I'd pick an empty array here, to make the return type the same for all variants of this method.